packages/ocephes/ocephes.0.0.1/opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/Ocephes"
+bug-reports: "https://github.com/rleonid/Ocephes/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/rleonid/Ocephes"
```

Comment:
If you could add `.git` to the end here, it would help opam understand that it needs to speak git to this URL in order to retrieve the dev version.